src/ai/backend/agent/stats.py

```diff
@@ -66,7 +68,9 @@ def check_cgroup_available():
 
 class StatModes(enum.Enum):
     CGROUP = "cgroup"
+    CGROUPFS = "cgroupfs"
     DOCKER = "docker"
+    SYSTEMD = "systemd"
 
     @staticmethod
     def get_preferred_mode():
```

Comment:
This function should be updated to auto-detect the cgroupfs/systemd modes as well.